package.json

```diff
@@ -3,10 +3,17 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@material-ui/core": "^1.3.1",
+    "@material-ui/icons": "^1.1.0",
+    "mdi-material-ui": "^5.2.0",
```

Comment:
whats this?

Reply:
@material-ui/core are for the components and the other 2 are icon packages

Reply:
cool sounds good.